library/ajax/sql_server_status.php

```diff
@@ -17,7 +17,7 @@
  * */
 
 $ignoreAuth = true;
-
+$GLOBALS['connection_pooling_off'] = true; // force off database connection pooling
```

Comment:
@sjpadgett ,
Not sure if this is needed here. Since it was in the sql_upgrade.php script figured you'd want it here also. (i noted this when debugging the `$GLOBALS['connection_pooling_off']` in the sql_upgrade.php script)